Sorts/test/SimplifiedWiggleSort.test.js

```diff
@@ -0,0 +1,29 @@
+import { simplifiedWiggleSort } from '../SimplifiedWiggleSort.js'
+
+describe('simplified wiggle sort', () => {
+  test('given array of symbols [\'b\', \'a\', \'c\'] return wiggle sorted array [\'a\', \'c\', \'b\']', () => {
```

Comment:
You can use a template string literal - or simply double quotes - here to avoid the need to escape all the strings. The test message probably shouldn't contain a serialized version of the data though, but instead just say what the test is supposed to check (something like "works properly with strings").